src/auth.cpp

```diff
@@ -104,7 +104,7 @@ int createAuthHeaderAKAv1MD5(const char *user,
 
 /* This function is from RFC 2617 Section 5 */
 
-void hashToHex (md5_byte_t *_b_raw, unsigned char *_h)
+static void hashToHex (md5_byte_t *_b_raw, unsigned char *_h)
```

Comment:
While you're here, you may as well remove the space between the function name and the parenthesis.